v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/templates/ProtegrityDataTokenization.java

```diff
@@ -216,7 +224,9 @@ public static void main(String[] args) {
    * @return The pipeline result.
    */
   public static PipelineResult run(ProtegrityDataTokenizationOptions options,
-      DataflowPipelineOptions dataflowOptions) {
+      String serviceAccount) {
```

Comment:
Could you please add this parameter to the docstring?

Reply:
Yes, of course. Done.